server/routes/ticket.ts

```diff
@@ -0,0 +1,39 @@
+import { Router } from 'express';
+import { TicketModel } from "../models/tickets/ticket.model";
+
+const router = Router();
+
+router.post('/', async (req, res, next) => {
```

Comment:
Try to properly type your functions like you would in Java (e.g. `import { Request, Response, NextFunction } from 'express';` at the top, then `async (req: Request, res: Response, next: NextFunction) => {}`)